go.sum

```diff
@@ -704,6 +704,8 @@ github.com/openshift/api v0.0.0-20230320211411-560b2fb170af h1:js8+NVUCTK7cz+IO6
 github.com/openshift/api v0.0.0-20230320211411-560b2fb170af/go.mod h1:ctXNyWanKEjGj8sss1KjjHQ3ENKFm33FFnS5BKaIPh4=
 github.com/openshift/aws-account-operator/api v0.0.0-20220921130557-58a04a5f1ba2 h1:qK7JrY2cBOKN/cDnHI6kRVHOIqMWHhkV7b3BQgldQ7A=
 github.com/openshift/aws-account-operator/api v0.0.0-20220921130557-58a04a5f1ba2/go.mod h1:1PdbQqTDrejSl9zsScM1x59f0oHNTsAgoJqTZqTkH/U=
```

Comment:
Changes make sense, but I think this needs a `go mod tidy` which will delete these two lines